frontend/components/LspAssets.tsx

```diff
@@ -102,49 +100,11 @@ export default function LspAssets() {
         </Text>
         {lsp7Assets.length === 0 ? (
           emptyLS7PAssets()
-        ) : (
-          <ul>
-            {lsp7Assets.map((asset, index) => (
-              <li key={index}>
-                <p>
-                  {asset.name} - {asset.address} - {asset.balance}
-                </p>
-                <MoveAssetToUpButton
-                  asset={asset.address!}
-                  from={graveVault!}
-                />
-              </li>
-            ))}
-          </ul>
-        )}
-      </Box>
-      <Box>
-        <Text
-          color="white"
-          fontWeight={400}
-          fontSize="16px"
-          fontFamily="Bungee"
-          mb="20px"
-        >
-          LSP8 Assets
-        </Text>
-        <Text
-          color="white"
-          fontWeight={400}
-          fontSize="16px"
-          fontFamily="Bungee"
-          mb="20px"
-          mt="20px"
-          textAlign="center"
-        >
-          Coming soon!
-        </Text>
-
-        {/* <ul>
-          {lps8Assets.map((asset, index) => (
-            <li key={index}>{asset}</li>
-          ))}
-        </ul> */}
+        ) :
+            lsp7Assets.map((asset, index) => (
+                    <LSP7Panel tokenName={asset.name!} tokenAmount={asset.balance!.toString()} tokenAddress={asset.address!} vaultAddress={graveVault!} />
```

Comment:
We need `Processing` state in the Revive button, where click is disabled until both transactions are done.